pkg/metrics/constants.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+const (
+	// OperatorSDKPrometheusMetricsPortValue defines the port which expose prometheus metrics
+	OperatorSDKPrometheusMetricsPort = 60000
+
+	// OperatorSDKPrometheusMetricsPortName define the port name used in kubernetes deployment and service
+	OperatorSDKPrometheusMetricsPortName = "sdk-metrics"
```

Comment:
Any reason to rename the port to `sdk-metrics`?
I'm thinking `metrics` is simple and clear enough. The sdk prefix doesn't add much.

Reply:
Yes, the reason being port names need to be unique in the deployment files and `metrics` is a very generic name, so if a user defines a new one the `metrics` port would not make sense any more. Plus in the next few PRs when the controller-runtime metrics gets into a release we want to add a port to the deployment and service resources and `8080` where they expose their metrics and we want to name it differently. Hope that makes sense?

Reply:
So we are going to serve two metrics endpoints one for the SDK and one for the controller runtime metrics? 

Could we just register the controller-runtime metrics as apart of the sdk metrics and then serve via a single endpoint?

Reply:
Yes for now, well currently there are no metrics in controller-runtime release so there are no metrics there yet. 
But see https://github.com/operator-framework/operator-sdk/pull/715#issuecomment-436928123 

> My plan is to adjust controller-runtime to not use global registry and we can then do a follow up PR to reuse that registry once that is merged.

Hope that clears things up?